docs/modules/servers/partials/configure/redis.adoc

```diff
@@ -47,3 +50,29 @@ io-threads-do-reads yes
 ....
 
 However, if your machine has only 1 CPU core or your Redis usage is not intensive, you will not benefit from this.
+
+== TLS/SSL
+
+To enable ssl mode, set `redis.ssl.enalbed` to true. The keyword `rediss` must be used in redisURL. For example:
+....
+redisURL=rediss://123@localhost:6379?verifyPeer=NONE
+....
+Once ssl mode is enabled, the following properties can be specified:
+
+|===
+| Property name | explanation
+
+| redis.ignoreCertificateCheck
+| To tell James to ignore any certificate check when connecting to redis via ssl. Defaults to true. If the value is false, the key store file would be used for the check.
+
+| redis.keystore.file.path
+| The file path pointing to the p12 file containing private key, cert and ca cert. Defaults to cacerts file path in java home location.
+
+| redis.keystore.password
+| The password of the p12 file. Defaults to empty.
+|===
+
+`verifyPeer=NONE` is used by Lettuce to verifies the certificate against the validity and the common name (Name validation not supported on Java 1.6, only available on Java 1.7 and higher) of the Redis host you are connecting to.
```

Comment:
Is it? That's not what I understand here => https://github.com/redis/lettuce/wiki/SSL-Connections